### ISSUE TITLE
[BUG] reactiveui.academy - domain experied

### ISSUE THREAD

**Issue by KiruyaMomochi** [reactiveui/website#653]
## TL;DR: Domain reactiveui.academy expired.

**Describe the bug**

The domain to Reactive Academy, which you can find on ReactiveUI Homepage, has expired.
The link is redirected from https://www.reactiveui.net/training to https://reactiveui.academy/.
And opening https://reactiveui.academy/ will show you a page that expired domain could have.

**Steps To Reproduce**
1. Open https://www.reactiveui.net/
2. Click [Training](https://www.reactiveui.net/training)

**Expected behavior**
A reactive academy website should be showed.

**Screenshots**

![image](https://user-images.githubusercontent.com/65301509/133860463-8663ed85-b6fe-4b37-9342-e989c66c9c0a.png)

**Environment**

- OS:       N/A
- Device:   N/A
- Version:   N/A
- Working Version: N/A
